shared/chat/conversation/input-area/normal/mention-input.js

```diff
@@ -219,27 +243,33 @@ class MentionInput extends React.Component<MentionInputProps, MentionState> {
 
   // End desktop only.
 
-  _forceSubmit = () => {
+  // If you type @notausername<enter> or #notachannel<enter>, we've essentially 'picked' nothing and really want to submit ... all the way out.
```

Comment:
sorry, the ellipsis here was meant to stand for the rest of the comment moved from `insertMention`:

`...really want to submit. This is a little wonky cause this component doesn't directly know if the list is filtered all the way out.`

Reply:
Major facepalm on my part. Got lost in the comment moving.